### DISCUSSION TITLE
Public Rally/Demo registry, map, calendar, gallery

### DISCUSSION THREAD

**Discussion by caffeine2022** [WeAreMahsaAmini/Core#289] [Ideas]
Basically calendar function with some attribute(address, timedate, ...) storage that organizer registers then can be viewed as map/gallery/list/...

Noticed people are already collecting date/time/city also connected it google map.
https://twitter.com/ps752justice/status/1575366030758744065?s=20&t=cTBlQKOQRdfzXIQkhSj7Hg

Could be a simple register interface that has basic features(duplication checking) CURD like simple app/webapp/service and UI/UX extended later on.

Can be used by organizers to plan better

Can create calendar event to be stored with address and time in calendar app

Could be further later extended with links to Live casts, Banners, Galleries, Hashtag's, Post to social media platforms

Can be associated with hashtags based on movement, city, ... 

Review/rating can be added as feedback for the organizer in case interested in retrospectives

**Comment by caffeine2022** [comment]
From the moment I shared the idea - this list increased from 77 to 130. And people are validating the need.

Join the Freedom Rally for Iran, on Saturday, Oct 1
In 77 cities around the world so far & still counting...

🗺Map
https://google.com/maps/d/edit?mid=1WX-p_1BVlIjywka1zsopdEQhNg1smTM&usp=sharing

📊Datasheet
https://docs.google.com/spreadsheets/d/14eXjKyeC7tD4d_K5lVSNpuiPgnrdb87QCGwNCLJDnkI/edit?usp=sharing

These pages are constantly getting updated...
Stay tuned ✌️

#MahsaAmini #OpIran #مهسا_امینی 
#PS752

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر
